playbooks/modules/info.yml

```diff
@@ -395,3 +395,50 @@
           - filter_key: "name"
             filter_operator: "equal"
             filter_value: "xxx"
+
+    - name: Get all writable snapshots from PowerScale cluster
+      dellemc.powerscale.info:
+        onefs_host: "{{ onefs_host }}"
+        verify_ssl: "{{ verify_ssl }}"
+        api_user: "{{ api_user }}"
+        api_password: "{{ api_password }}"
+        gather_subset:
+          - writable_snapshot
+
+    - name: To get the specific writable snapshot.
+      dellemc.powerscale.info:
+        onefs_host: "{{ onefs_host }}"
+        verify_ssl: "{{ verify_ssl }}"
+        api_user: "{{ api_user }}"
+        api_password: "{{ api_password }}"
+        gather_subset:
+          - writable_snapshot
+        query_parameters:
+          writable_snapshot:
+            wspath: "/ifs/test_mkdir"
+
+    - name: To filter the writable snapshot in ascending order.
+      dellemc.powerscale.info:
+        onefs_host: "{{ onefs_host }}"
+        verify_ssl: "{{ verify_ssl }}"
+        api_user: "{{ api_user }}"
+        api_password: "{{ api_password }}"
+        gather_subset:
+          - writable_snapshot
+        query_parameters:
+          writable_snapshot:
+            dir: ASC
+            limit: 1
+
+    - name: To sort the writable snapshot in ascending order.
```

Comment:
Please update the name of the task query parameter is sort

Reply:
done